src/main/webapp/person/visualizePersonalInfo.jsp

```diff
@@ -40,7 +40,7 @@
 </h2>
 
 <logic:messagesPresent message="true">
-	<html:messages id="messages" message="true" bundle="APPLICATION_RESOURCES">
+	<html:messages id="messages" message="true" bundle="ACADEMIC_OFFICE_RESOURCES">
```

Comment:
In this case it would make more sense to add the missing label in the APPLICATION_RESOURCES, (or any other of the scope of this interface), rather than having a functionality related the user's own information dependent on academic office resource bundles.